packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -379,22 +402,30 @@ export class AppStudioPluginImpl {
     } else {
       manifest = await this.createManifest(ctx.projectSettings!);
     }
+    // await fs.writeFile(
+    //   `${ctx.root}/.${ConfigFolderName}/${REMOTE_MANIFEST}`,
+    //   JSON.stringify(manifest, null, 4)
+    // );
     await fs.writeFile(
-      `${ctx.root}/.${ConfigFolderName}/${REMOTE_MANIFEST}`,
+      `${ctx.root}/${AppPackageFolderName}/${REMOTE_MANIFEST}`,
```

Comment:
Make sure the AppPackageFolder is created.

Reply:
Thanks for the comments. I have added the directory checking in fx-core/src/core/index.ts